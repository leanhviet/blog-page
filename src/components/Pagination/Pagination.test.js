// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { PaginationBase } from './index'

const props = {
  totalRecords: 30,
  pageLimit: 5,
  initialPage: 1,
  pagesToShow: 5,
}

describe('PaginationBase component', () => {
  const wrapper = shallow(<PaginationBase />)
  const wrapperWithProps = shallow(<PaginationBase {...props} />)

  it('should be defined PaginationBase component', () => {
    expect(wrapper).toBeDefined()
  })

  // Testing Snapshot
  it('should be render snapshot PaginationBase component', () => {
    expect(wrapper).toMatchSnapshot()
    expect(wrapperWithProps).toMatchSnapshot()
  })
})
