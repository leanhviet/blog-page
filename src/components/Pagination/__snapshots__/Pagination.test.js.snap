// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationBase component should be render snapshot PaginationBase component 1`] = `""`;

exports[`PaginationBase component should be render snapshot PaginationBase component 2`] = `""`;
